File: HappyB.java

public class HappyB {
	static String vseNej = "Všechno nejlepší!";
	
	private static String HappyB(String jmeno) {
		return jmeno;		
	}
	
	private static String rekni(String jmeno, String hlaska) {
		int stopka = 0;
		
		while (true) {
			System.out.println(HappyB(jmeno) + " " + HappyB.vseNej );
			stopka++;
			if (stopka == 30) {
				break;
			}
		}
		
		return HappyB.za_rok();
	}
	
	private static String za_rok() {
		String za_rok = "Tak zase za rok!";
		return za_rok;
	}
	
	public static void main(String[] args) {
		rekni("sestøièko", vseNej);
	}
}

File: happyb.py
class HappyB(object):
    vseNej = "Všechno nejlepší!\n"

    def __init__(self, jmeno):
        self.jmeno = jmeno

    def rekni(self, hlaska):
        stopka = 0
        while (True):
            print("{jmeno}: {hlaska}".format(jmeno=self.jmeno, hlaska=hlaska))
            stopka = stopka + 1
            if stopka == 30:
                break
        return HappyB.za_rok()

    def za_rok():
        return "Tak zase za rok!"

k = HappyB("Sestřičko")

print(k.rekni(HappyB.vseNej))
